### PR TITLE
Update files with PEP8 standards in folder dojo/tools #006

### PR DESCRIPTION
**Pull request by ajtortolero**
Update files with PEP8 standars in folders dojo/tools/

Folder affected
dojo/tools/sarif
dojo/tools/scantist
dojo/tools/scout_suite
dojo/tools/semgrep
dojo/tools/skf
dojo/tools/snyk
dojo/tools/solar_appscreener
dojo/tools/sonarqube
dojo/tools/sonatype
dojo/tools/spotbugs
dojo/tools/ssl_labs
dojo/tools/sslscan
dojo/tools/sslyze
dojo/tools/stackhawk
dojo/tools/talisman
dojo/tools/tenable
dojo/tools/terrascan
dojo/tools/testssl
dojo/tools/tfsec
dojo/tools/trivy
dojo/tools/trivy_operator
dojo/tools/trufflehog
dojo/tools/trufflehog3
dojo/tools/trustwave
dojo/tools/trustwave_fusion_api
dojo/tools/twistlock
dojo/tools/vcg
dojo/tools/veracode
dojo/tools/veracode_sca
dojo/tools/wapiti
dojo/tools/wazuh
dojo/tools/wfuzz
dojo/tools/whispers
dojo/tools/whitehat_sentinel
dojo/tools/whitehat_sentinel
dojo/tools/whitesource
dojo/tools/wpscan
dojo/tools/xanitizer
dojo/tools/yarn_audit
dojo/tools/zap

### CODE REVIEW

**Review by Maffooch — CHANGES_REQUESTED** [review]

File: dojo/tools/scout_suite/parser.py [hunk 1]
```diff
@@ -127,8 +159,8 @@ def formatview(self, depth):
         else:
             return ""
 
-    def recursive_print(self, src, depth=0, key=''):
-        tabs = lambda n: ' ' * n * 2
+    def recursive_print(self, src, depth=0, key=""):
+        def tabs(n): return " " * n * 2
```
[the remark this answers is not in the document]
Comment (ajtortolero) [reply dojo/tools/scout_suite/parser.py:163]: Hi @Maffooch, flake8 applies this change to comply with this [rule](https://www.flake8rules.com/rules/E731.html).

File: dojo/tools/snyk/parser.py [hunk 2]
```diff
@@ -34,121 +32,149 @@ def parse_json(self, json_output):
         try:
             data = json_output.read()
             try:
-                tree = json.loads(str(data, 'utf-8'))
-            except:
+                tree = json.loads(str(data, "utf-8"))
+            except BaseException:
                 tree = json.loads(data)
-        except:
+        except BaseException:
```
[the remark this answers is not in the document]
Comment (ajtortolero) [reply dojo/tools/snyk/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8319/commits/c5c7c4c68a0296160dac914cdd2177313507dcf4

File: dojo/tools/snyk/parser.py [hunk 3]
```diff
@@ -34,121 +32,149 @@ def parse_json(self, json_output):
         try:
             data = json_output.read()
             try:
-                tree = json.loads(str(data, 'utf-8'))
-            except:
+                tree = json.loads(str(data, "utf-8"))
+            except BaseException:
```
[the remark this answers is not in the document]
Comment (ajtortolero) [reply dojo/tools/snyk/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8319/commits/c5c7c4c68a0296160dac914cdd2177313507dcf4

File: dojo/tools/scout_suite/parser.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Maffooch) [reply dojo/tools/scout_suite/parser.py:163]: Thanks for the info!

**Review by Maffooch — APPROVED** [review]

**Review by mtesauro — APPROVED** [review]
Approved

**Review by grendel513 — APPROVED** [review]

File: dojo/tools/scout_suite/parser.py [hunk 4]
```diff
@@ -8,7 +7,7 @@
 
 
 class ScoutSuiteParser(object):
-    """"ScoutSuite Wiki: https://github.com/nccgroup/ScoutSuite/wiki"""
+    """ "ScoutSuite Wiki: https://github.com/nccgroup/ScoutSuite/wiki"""
```
Comment (cneill) [inline dojo/tools/scout_suite/parser.py:10]: The `"` before `ScoutSuite` seems unnecessary

File: dojo/tools/wfuzz/parser.py [hunk 5]
```diff
@@ -29,13 +29,12 @@ def get_description_for_scan_types(self, scan_type):
...
 
     def get_findings(self, filename, test):
-
         data = json.load(filename)
 
         dupes = {}
         for item in data:
             url = hyperlink.parse(item["url"])
-            payload = item["payload"]
+            item["payload"]
```
Comment (cneill) [inline dojo/tools/wfuzz/parser.py:37]: This seems unnecessary - is there a reason for removing the variable name here?

**Review by cneill — CHANGES_REQUESTED** [review]
Just a few things I noticed

File: dojo/tools/scout_suite/parser.py (hunk 4 above)
Comment (ajtortolero) [reply dojo/tools/scout_suite/parser.py]: I removed blamk space in commit https://github.com/DefectDojo/django-DefectDojo/pull/8319/commits/1053acfe10f196c6dc7c94ff932aaff110e8fa17

**Review by cneill — APPROVED** [review]

**Pull request merged**